### ISSUE TITLE
Localize new Webmaker.org site

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-desktop#181]
beta.webmaker.org will soon replace webmaker.org. Localize anything that hasn't already been localized. 

cc @secretrobotron

**Comment by xmatthewx** [comment]
How's this lookin'? Are we prepped for localization?

**Comment by secretrobotron** [comment]
My guess is no. @alicoding it out today — he's been head down in setting up localization for the app.
Would be good for him to give us some instruction on Monday.

@mmmavis would you be able to point at the stuff that needs to change on that beta page (i.e. where we need to add i18n stuff)?

**Comment by alicoding** [comment]
I haven't look at how beta page is built but if it is using React then it
can be done quick and probably half of my day on Monday otherwise I have to
look and report back on Monday (or when I get home)

**Comment by secretrobotron** [comment]
There he is. :+1: thanks @alicoding.

**Comment by secretrobotron** [comment]
@alicoding: yep, react: https://github.com/mozilla/webmaker-browser/blob/develop/src/pages/splash/splash.jsx

Should be straightforward. Just need to get something up on transifex ASAP. Should we use YAML?

**Comment by secretrobotron** [comment]
Related to https://github.com/mozilla/webmaker-core/issues/148

**Comment by alicoding** [comment]
Yep let's use YAML for now and pull them to the repo manually once they are
all translated

**Comment by xmatthewx** [comment]
What's the status on this?

**Comment by alicoding** [comment]
I've localized this page in English https://github.com/mozilla/webmaker-browser/blob/develop/src/locales/en-US.yaml but it still needs to be pull down from Transifex for whatever language we want to enable and that should be it :)

**Comment by xmatthewx** [comment]
Is this something you're able to work on this week @alicoding? Or is your time locked on another project? cc @k88hudson

**Comment by alicoding** [comment]
@xmatthewx I would rather document the steps or show other people how it's done than doing it myself, so that they know how to do it?

**Comment by alicoding** [comment]
Going to make a PR soon, but just updated the resource file again since we added so many later and it is out-dated now. https://www.transifex.com/webmaker/webmaker/betawebmakerorg/

**Comment by xmatthewx** [comment]
Thanks @alicoding! Yer the best.

**Comment by xmatthewx** [comment]
cc @thisandagain

**Comment by alicoding** [comment]
Bangali and Indonesian needs to be updated on Transifex since they are not 100% translated yet, so this is a bit blocked.
